src/js/media-poster-image.js

```diff
@@ -53,7 +74,10 @@ class MediaPosterImage extends window.HTMLElement {
     this.attachShadow({ mode: 'open' });
     this.shadowRoot.appendChild(template.content.cloneNode(true));
 
+    this.container = this.shadowRoot.querySelector('#container');
     this.image = this.shadowRoot.querySelector('#image');
+
+    this.image.addEventListener('load', () => loadFade(this.image));
```

Comment:
@cjpillsbury would probably know the answer here (I genuinely don't), do we need to remove event listeners like this that get attached in the constructor

Reply:
yeah, they should get removed. Though, also can use `{once: true}` option to auto-remove it when `load` triggers.

Reply:
Can't use `{ once: true }` since it's valid to change the `media-poster-image` src.

Reply:
ah, that's true. I guess just need to make sure it's removed on disconnectedCallback then.